src/.bazelrc

```diff
@@ -74,9 +65,6 @@ build:windows_env --noincompatible_enable_cc_toolchain_resolution
 # Android specific options
 build:android_env --copt "-DOS_ANDROID"
 build:android_env --build_tag_filters=-noandroid
-test:android_env   --test_tag_filters=-noandroid
-
-## Android Blzmod specific options
-### --experimental_google_legacy_api is required for all platforms.
-build:bzlmod --experimental_google_legacy_api
 build:android_env --experimental_enable_android_migration_apis
+build:android_env --experimental_google_legacy_api
```

Comment:
`--experimental_google_legacy_api` was required for all platforms when I checked last time.
Would you please double check it?

Reply:
Looks this is no longer necessary in practice.

Let's see if we can merge the following commit first.

* https://github.com/google/mozc/pull/1119